app/jobs/wca_cronjob.rb

```diff
@@ -116,5 +116,21 @@ def reset_error_state!
         last_error_message: nil,
       )
     end
+
+    def serialize
```

Comment:
Why do you need this custom serialization logic? `my_cronjob_class.cronjob_statistics` gives you a plain normal AR model, which you can use for plain normal `as_json` and `serializable_hash` shenanigans

Reply:
(ironically you're including `cronjob_statistics` in your serialization response, whereas most of the other methods like `in_progress`, `scheduled`, `finished` and many others are just `delegate`d to that same `cronjob_statistics` entity.)